server/game/cards/characters/04/robbstark.js

```diff
@@ -0,0 +1,58 @@
+const DrawCard = require('../../../drawcard.js');
+
+class RobbStark extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => (
+                this.game.currentChallenge &&
+                this.game.currentChallenge.challengeType === 'military'
+            ),
+            match: this,
+            effect: ability.effects.modifyStrength(this.numberOfLoyalChars())
```

Comment:
This will need to use the dynamic strength effect. Currently what this will do is set the strength modifier to be the number of loyal cards in play at the start of the game, which is 0.

```javascript
effect: ability.effects.dynamicStrength(() => this.numberOfLoyalChars())
```

This will recalculate the strength modifier every time characters enter / leave play and periodically.